src/extension.ts

```diff
@@ -134,11 +140,71 @@ export async function activate(context: vscode.ExtensionContext) {
 	const intuitaTextDocumentContentProvider =
 		new IntuitaTextDocumentContentProvider();
 
+	// @TODO split this large file to modules
+
+	/**
+	 * User
+	 */
 	const globalStateAccountStorage = new GlobalStateAccountStorage(
 		context.globalState,
 	);
 	const userService = new UserService(globalStateAccountStorage);
 
+	const intuitaWebviewProvider = new IntuitaPanel(context);
+	const view = vscode.window.registerWebviewViewProvider(
+		'intuita-webview',
```

Comment:
can you check if it works with `vsce package` command?

Reply:
works fine

Reply:
@grzpab i've also made updates to `vscodeignore`, excluded node_module form webview